packages/batch/tests/unit/AsyncBatchProcessor.test.ts

```diff
@@ -0,0 +1,237 @@
+/**
+ * Test AsyncAsyncBatchProcessor class
+ *
+ * @group unit/batch/class/asyncAsyncBatchProcessor
```

Comment:
Should this have the tests for Lambda context as well?

Reply:
Added the tests!